src/base/select-element.ts

```diff
@@ -97,8 +115,11 @@ export class SelectElement extends SgdsFormValidatorMixin(DropdownListElement) i
 
   connectedCallback(): void {
     super.connectedCallback();
-    this.addEventListener("blur", async () => {
-      this.invalid = this.menuIsOpen ? false : !this._mixinReportValidity();
+    this.addEventListener("blur", async e => {
```

Comment:
doing this so that when the user traverse the menu, the component wont check for its validity as its still considered self